### PR TITLE
[IOTDB-3280] Can not set ttl to all sg in cli with one sql

### PR DESCRIPTION
**Pull request by 23931017wu**
### After
image
image

### CODE REVIEW

**Review by MarcosZyk — COMMENTED** [review]
Please update User-Guide doc.

**Comment by 23931017wu** [comment]
> Please update User-Guide doc.

OK~

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java [hunk 1]
```diff
@@ -77,6 +77,31 @@ public void testTTL() throws SQLException, InterruptedException {
...
+        statement.execute("SET TTL TO root.TTL_SG2.s1 1000");
+      } catch (SQLException e) {
+        assertEquals(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode(), e.getErrorCode());
+      }
+
+      try {
+        statement.execute("SET TTL TO root.** 1000");
+      } catch (SQLException e) {
+        assertEquals(TSStatusCode.TIMESERIES_NOT_EXIST.getStatusCode(), e.getErrorCode());
+      }
```
Comment (MarcosZyk) [inline integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java:92]: The IT in integration package is only run by old-standalone. Add the IT to integration-test package and follow the new-IT rule. Besides, if you expect exception in this test case, ```Assert.fail()``` is necessary.

**Review by neuyilan — APPROVED** [review]
LGTM

**Review by MarcosZyk — COMMENTED** [review]
Only one IT problem left. PTAL. 

Besides, the feature improvement should be updated in ```docs/UserGuide/Operate-Metadata/TTL.md``` and ```zh/docs/UserGuide/Operate-Metadata/TTL.md```. Please add some sql examples.

File: integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java (hunk 1 above)
Comment (23931017wu) [reply integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java:92]: Okay, got it.

**Comment by 23931017wu** [comment]
> Only one IT problem left. PTAL.
> 
> Besides, the feature improvement should be updated in `docs/UserGuide/Operate-Metadata/TTL.md` and `zh/docs/UserGuide/Operate-Metadata/TTL.md`. Please add some sql examples.

One more problem: When ConfigNode sends a setTTLReq to the Datanode, it now sends a setTTLReq to each storage group, so I will change this to send all storage groups to the Datanode as well

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Review by MarcosZyk — APPROVED** [review]

**Pull request merged**